pkg/bucket/bandwidth/throttler.go

```diff
@@ -0,0 +1,115 @@
+/*
+ * MinIO Cloud Storage, (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package bandwidth
+
+import (
+	"sync"
+	"sync/atomic"
+	"time"
+)
+
+// Throttler implements the throttling for bandwidth
+type Throttler struct {
```

Comment:
```suggestion
type Throttle struct {
```
Typically 'er' means an interface and the throttle on a car is also called a throttle. `bandwidth.NewThrottle(...)` looks perfectly fine.

Reply:
Hmm.. Yup, will change it